etc/ci/buildbot_steps.yml

```diff
@@ -63,6 +63,14 @@ mac-rel-css2:
   - ./mach test-wpt --release --processes 4 --total-chunks 6 --this-chunk 6 --log-raw test-wpt.log --log-errorsummary wpt-errorsummary.log --always-succeed
   - ./mach filter-intermittents wpt-errorsummary.log --log-intermittents intermittents.log --log-filteredsummary filtered-wpt-errorsummary.log --tracker-api default --reporter-api default
 
+mac-rel-bluetooth:
```

Comment:
Instead of a different job, please add this to an existing mac job that runs WPT tests.

Reply:
Ok, I've moved the execution to `mac-rel-wpt4`